README.md

```diff
@@ -231,8 +231,8 @@ If a given `Animal` has no children, its `child_names` list is empty.
 #### Constraints and Rules
 - `@fold` can only be applied to vertex fields, except the root vertex field.
 - May not exist at the same vertex field as `@recurse`, `@optional`, or `@output_source`.
-- Expanding vertex fields is not allowed within a scope marked `@fold`.
-- `@tag` and `@fold` may not be used within a scope marked `@fold`.
+- Only one vertex field may be expanded within a scope marked `@fold`. Note that this applies for all nested scopes within.
```

Comment:
I think this wording will confuse people -- writing precise documentation is hard. The limitation is that "no scope within a scope marked `@fold` may contain more than one expanded vertex field", but ideally we'd also have an example of what is okay and what is not.